core/src/main/java/org/infinispan/configuration/cache/PersistenceConfiguration.java

```diff
@@ -13,17 +13,26 @@
  */
 public class PersistenceConfiguration implements Matchable<PersistenceConfiguration> {
    public static final AttributeDefinition<Boolean> PASSIVATION = AttributeDefinition.builder("passivation", false).immutable().build();
+   public static final AttributeDefinition<Integer> AVAILABILITY_INTERVAL = AttributeDefinition.builder("availabilityInterval", 1000).immutable().build();
+   public static final AttributeDefinition<Integer> CONNECTION_ATTEMPTS = AttributeDefinition.builder("connectionAttempts", 10).immutable().build();
+   public static final AttributeDefinition<Integer> CONNECTION_INTERVAL = AttributeDefinition.builder("connectionInterval", 50).immutable().build();
    static AttributeSet attributeDefinitionSet() {
-      return new AttributeSet(PersistenceConfiguration.class, PASSIVATION);
+      return new AttributeSet(PersistenceConfiguration.class, PASSIVATION, AVAILABILITY_INTERVAL, CONNECTION_ATTEMPTS, CONNECTION_INTERVAL);
    }
 
    private final Attribute<Boolean> passivation;
+   private final Attribute<Integer> availabilityInterval;
```

Comment:
This should probably be a long I would think.

Reply:
Isn't Integer.MAX_VALUE equivalent to over 3 weeks in milliseconds?